src/full/Agda/Syntax/Concrete.hs

```diff
@@ -184,7 +184,16 @@ type OpAppArgs' e = [NamedArg (MaybePlaceholder (OpApp e))]
 
 -- | Concrete patterns. No literals in patterns at the moment.
 data Pattern
-  = IdentP QName                           -- ^ @c@ or @x@
+  = IdentP Bool QName                      -- ^ @c@ or @x@
+                                           --
+                                           -- If the boolean is
+                                           -- 'False', then the
+                                           -- 'QName' must not refer
+                                           -- to a constructor or a
+                                           -- pattern synonym. The
+                                           -- value 'False' is used
+                                           -- when a hidden argument
+                                           -- pun is expanded.
```

Comment:
Weird formatting.
A line length of 100 is acceptable.

Reply:
When I have two buffers of equal width open next to each other in a full-screen Emacs window on my laptop (with a certain screen resolution and a certain font/font size) the buffer widths are about 72 columns.